src/track/trace.h

```diff
@@ -59,7 +89,7 @@ struct Trace
     bool fill(int skip)
     {
         int size = unw_backtrace(m_data, MAX_SIZE);
```

Comment:
backtrace::fill(m_data, MAX_SIZE) or similar, which gets implemented in both, libunwind or your new custom backtrace lib